storm-core/src/jvm/org/apache/storm/scheduler/Cluster.java

```diff
@@ -103,6 +103,9 @@ public Cluster(Cluster src) {
         this.status.putAll(src.status);
         this.topologyResources.putAll(src.topologyResources);
         this.blackListedHosts.addAll(src.blackListedHosts);
+        if (src.networkTopography != null) {
```

Comment:
is this supposed to be == null. why are we creating new Map if there is one already.